app/utils/stacking/pox.spec.ts

```diff
@@ -58,6 +58,7 @@ test.skip('making a lock-stx transaction', async () => {
   });
   await waitForTxConfirm(`0x${lockTxid}`);
   const stackerInfo = await client.getStackerInfo(address);
+  if ('error' in stackerInfo) return;
```

Comment:
Since this is a test, maybe first add a test that there is no error, and if so, add an exception / return here?

Reply:
This was just to satisfy types. Test is skipped and this code will be replaced by Ken's stacking lib soon.